pmpro-network.php

```diff
@@ -245,25 +255,48 @@ function pmpron_extra_page_settings($pages) {
 }
 add_action('pmpro_extra_page_settings', 'pmpron_extra_page_settings');
 
+/**
+ * Get the ID (or post object) for the "Manage Sites" page.

```

Comment:
We should choose whether we are returning the ID or the post object. Otherwise, anywhere that calls this function will need to account for both options